src/app/auth/index.js

```diff
@@ -15,9 +17,9 @@ function parseEvent(event) {
 exports.handler = async (event, context) => {
     try {
         const params = parseEvent(event);
-        return await handleRequest(params.cookies, params.code, params.state, dynamoDBClient);
+        return await handleRequest(params.cookies, params.code, params.state, dynamoDBClient, apiClient);
```

Comment:
Wat is er nu gebeurt met `handleRequest`? Moet dat niet ook in deze PR?

Reply:
Goeie vraag. HandleRequest had al een optie om een apiClient mee te geven (ik geloof omdat we dat in tests al deden).

Reply:
Gek dat het dan ooit heeft gewerkt, of maar de `BrpApi` zelf een client aan als er geen mee wordt gegeven?

Reply:
Klopt, als er geen is meegegeven maakt hij er zelf 1 aan. Dat heeft als nadeel dat dat elke request gebeurde (incl. ophalen secrets e.d.). Deze route zorgt hopelijk voor snellere uitvoering.